rust-toolchain.toml

```diff
@@ -1,4 +1,4 @@
 [toolchain]
-channel = "1.63"
+channel = "1.74"
```

Comment:
The one open question on this: should we use `"stable"` here instead?

Reply:
I think keeping it set to a specific version is useful for reproducibility later on. That said, we can likely be more liberal with updating when we don't have issues with compatibility.